db/schema.rb

```diff
@@ -55,6 +55,33 @@
   add_index "identities", ["session_uuid"], name: "index_identities_on_session_uuid", unique: true, using: :btree
   add_index "identities", ["user_id"], name: "index_identities_on_user_id", using: :btree
 
+  create_table "piis", force: :cascade do |t|
```

Comment:
alas `attributes` is an ActiveRecord built-in method. http://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods.html#method-i-attributes

`fact_sets`? `user_facts`?